### PR TITLE
blog(embertimes161): template-colocation-migrator

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2020-08-21-the-ember-times-issue-161.md [hunk 1]
```diff
@@ -9,19 +9,39 @@ responsive: true
...
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Release of ember-component-template-colocation-migrator 1.1.0 🎉](https://github.com/ember-codemods/ember-component-template-colocation-migrator/releases/tag/v1.1.0)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+The [ember-component-template-colocation-migrator](https://github.com/ember-codemods/ember-component-template-colocation-migrator) codemod will co-locate component JS and HBS files as described in the [Component Templates Co-location RFC](https://emberjs.github.io/rfcs/0481-component-templates-co-location.html). The v1.1.0 release adds [support for migrating addons](https://github.com/ember-codemods/ember-component-template-colocation-migrator/pull/14) although templates shared by multiple components are excluded. In this case, it is up to end users how to list those; they are reported in the same way other skipped templates (like partials) are. Similarly Mixins which use layout are not migrated.
```
Comment (ijlee2) [inline source/2020-08-21-the-ember-times-issue-161.md]: I came up with a few suggestions:

- "will co-locate" -> "co-locates"
- "although" -> "but"
- "Mixins which" -> "mixins that"

```suggestion
The [ember-component-template-colocation-migrator](https://github.com/ember-codemods/ember-component-template-colocation-migrator) codemod co-locates component JS and HBS files as described in the [Component Templates Co-location RFC](https://emberjs.github.io/rfcs/0481-component-templates-co-location.html). The v1.1.0 release adds [support for migrating addons](https://github.com/ember-codemods/ember-component-template-colocation-migrator/pull/14), but templates shared by multiple components are excluded. In this case, it is up to end users how to list those; they are reported in the same way other skipped templates (like partials) are. Similarly, mixins that use layout are not migrated.
```

File: source/2020-08-21-the-ember-times-issue-161.md [hunk 2]
```diff
@@ -9,19 +9,39 @@ responsive: true
...
+
+```sh
+your-addon/
+  addon/
+    components/
+      some-component.js
+      some-component.hbs
+```
+
+Try it out today by running the migrator in your app: `npx github:ember-codemods/ember-component-template-colocation-migrator`
```
Comment (ijlee2) [inline source/2020-08-21-the-ember-times-issue-161.md]:
```suggestion
Try it out today by running the migrator in your app or addon: `npx github:ember-codemods/ember-component-template-colocation-migrator`
```

**Review by ijlee2 — APPROVED** [review]
Thanks for the write-up! Looks great!

**Comment by amyrlam** [comment]
nice one 🙌, thank you @chrisrng and @ijlee2!

**Pull request merged**